app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  def new
+    super
+  end
+
+  def create
+    super
+  end
+
+  def update
+    super
+  end
+
+  private
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])
+  end
```

Comment:
You dont need to overide devise controller you can just allow it in application controller